docs/src/main/sphinx/sql/drop-materialized-view.rst

```diff
@@ -0,0 +1,36 @@
+======================
+DROP MATERIALIZED VIEW
+======================
+
+Synopsis
+--------
+
+.. code-block:: text
+
+    DROP MATERIALIZED VIEW [ IF EXISTS ] view_name;
+
+Description
+-----------
+
+Drop an existing materialized view.
+
+The optional ``IF EXISTS`` clause causes the error to be suppressed if
```

Comment:
Would love a specific error message or code here if possible, rather than just "the error". Better for SEO.

Reply:
Not sure what the message is and hard to keep in sync. Leaving as is for now as this is a V1 doc set only